extensions/2.0/Khronos/KHR_materials_iridescence/README.md

```diff
@@ -0,0 +1,148 @@
+|------------------------------|---------------------------------------------------------------------|--------------------------------------------------------------------------------------------------|-----------------------|
+|**iridescenceFactor**         | `number`                                                            | The iridescence intensity.                                                                       | No, default: `0.0`    |
+|**iridescenceTexture**        | [`textureInfo`](https://www.khronos.org/registry/glTF/specs/2.0/glTF-2.0.html#reference-textureinfo) | The iridescence intensity stored in the red channel of the texture.                                                               | No                    |
+|**iridescenceIOR**               | `number`                                                            | The index of refraction of the dielectric thin-film layer.                                       | No, default: `1.8`    |
+|**iridescenceThicknessMinimum**  | `number`                                                            | The minimum thickness of the thin-film layer given in nanometers (nm).                                                    | No, default: `400.0`  |
+|**iridescenceThicknessMaximum**  | `number`                                                            | The maximum thickness of the thin-film layer given in nanometers (nm).                                                    | No, default: `1200.0` |
+|**iridescenceThicknessTexture**  | [`textureInfo`](https://www.khronos.org/registry/glTF/specs/2.0/glTF-2.0.html#reference-textureinfo) | The green channel of this texture defines the thickness of the thin-film layer by blending between the minimum and maximum thickness. | No                    |
+
+The values for iridescence intensity can be defined using a factor, a texture, or both.
+`iridescenceFactor` is multiplied with the red channel of `iridescenceTexture` to control the overall strength of the iridescence effect. If the texture is not set, a value of 1.0 is assumed. 
+```
+iridescence = iridescenceFactor * iridescenceTexture.r
+```
+
+If `iridescenceFactor` is zero (default), the iridescence extension has no effect on the material. 
+All textures in this extension use a single channel in linear space. 
+The thickness of the thin-film is set to `iridescenceThicknessMaximum` if `iridescenceThicknessTexture` is not given.
+If `iridescenceThicknessTexture` is set, the thickness of the thin-film varies between  `iridescenceThicknessMinimum` and `iridescenceThicknessMaximum` as follows: 
+```
+thickness = mix(iridescenceThicknessMinimum, iridescenceThicknessMaximum, iridescenceThicknessTexture.g)
+```
+The thin-film layer can have a different IOR than the underlying material. With `iridescenceIOR` one can set an IOR value for the thin-film layer independently.
```

Comment:
Where does the thin film layer belong in the layering stack? How does it interact with other extensions like clearcoat and sheen?

Reply:
@UX3D-kanzler @UX3D-becher @UX3D-eckerlein @PascalSchoen 
Can you please add the missing details depneding on the current implementation:
https://github.com/KhronosGroup/glTF-Sample-Viewer/pull/352

Reply:
@proog128 clearcoat is defined in the documentation. For sheen: it should have the same behaviour as with the default specular highlight as it's just a replacement for or a blend between it.

Reply:
@proog128 clearcoat is defined in the documentation. For sheen: it should have the same behaviour as with the default specular highlight as it's just a replacement for or a blend between it.